### ISSUE TITLE
Enhancement: optimise barcode scanning page for barcode scanning workflow

### ISSUE THREAD

**Issue by quimnut** [geoffhumphrey/brewcompetitiononlineentry#1338]
**Is your enhancement or feature request related to a problem? Please describe.**

In the barcode check in page where we register the entry and add a judging number, there are 2 additional fields - box and paid. With over 500 entries I don't use box #'s and just sort in to crates by category so i can leave it blank.

Because we're using barcode scanners here I expect I can just scan a pre-printed barcode that will tab space and move on to the next row.

However the paid button automatically moves to next field, but using a barcode scanner that is terminating with a tab, so i have to stop using the barcode scanner and click the radio button with a mouse for every entry.

Additionally having to update the paid status for every bottle forces this task into the registration process where it does not need to be, we can just look up entries page to see what wasn't paid. We're trying to use the barcode scanner to speed up a time consuming process but now we're checking if things are paid based off the label too.

**Describe the solution you'd like**

Have an option to remove the box and paid fields so the workflow is just 2 scans, entry no, judging no. Realize the benefit of the barcode scanner.

**Describe alternatives you've considered**

**Additional context**

**Comment by geoffhumphrey** [comment]
Good suggestion. And a pretty quick one to implement, fortunately. If you update the code in /admin/barcode_check-in.admin.php with [this commit,](https://github.com/geoffhumphrey/brewcompetitiononlineentry/blob/master/admin/barcode_check-in.admin.php) it should be good to go for version 2.3.X+.

**Comment by jimmyorkin** [comment]
I ended up just replacing the barcode checking modules for my competitions. I changed the tab order to what I wanted. Let me document the additional checks I make that finds errors that occur at login:

1. An attempt to reuse a Judging Number. It's happened. The row of Judging number stickers ended up on two entries.
2. Entry number not found.
3. Entries already assigned Judging numbers. This check looks at the judging numbers. BCOEM never uses zeros in the preset judging number. Our judging number always start with "00".
4. Entry already checked in.

Food for thought.
